zerver/actions/message_edit.py

```diff
@@ -248,7 +248,13 @@ def get_mentions_for_message_updates(message_id: int) -> Set[int]:
             flags=~UserMessage.flags.historical,
         )
         .filter(
-            Q(flags__andnz=(UserMessage.flags.mentioned | UserMessage.flags.wildcard_mentioned))
+            Q(
+                flags__andnz=(
+                    UserMessage.flags.mentioned
+                    | UserMessage.flags.wildcard_mentioned
+                    | UserMessage.flags.topic_wildcard_mentioned
+                )
+            )
```

Comment:
Should this be a function defined in `models.py` like the `where_starred` batch of functions? If so, we could call it `where_any_mentioned`, or something.

(The benefit of putting it on `models.py` after some testing is that we'd be providing an interface that more clearly promises that we're using a query that actually uses the index we have, which is much better for future code reuse -- though we would want to test the actual query with `EXPLAIN ANALYZE` to be certain)

Reply:
https://chat.zulip.org/#narrow/stream/3-backend/topic/topic_wildcard_mentioned.20flag/near/1673984